app/src/main/java/org/xbmc/kore/provider/MediaDatabase.java

```diff
@@ -346,6 +354,7 @@ public void onCreate(SQLiteDatabase db) {
                    MediaContract.SyncColumns.UPDATED + " INTEGER NOT NULL," +
                    MediaContract.SongsColumns.HOST_ID + " INTEGER NOT NULL " + References.HOST_ID + ", " +
                    MediaContract.SongsColumns.ALBUMID + " INTEGER NOT NULL, " +
+                   MediaContract.SongsColumns.ARTISTID + " INTEGER NOT NULL, " +
```

Comment:
Didn't explicitly test when a song has multiple artists. So, if I understand correctly, this can happen when a song is part of an album that has multiple artists, right?